cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
```

Comment:
We don't need this commonLabels configuration here; this is applied to all resources in the cluster overlays.

Reply:
Sure @larsks , I have removed it.

Reply:
@larsks does this resolve the CR?